src/_langs/en/updates/2015-07-22-mediastream-deprecations.markdown

```diff
@@ -0,0 +1,155 @@
+---
+
+layout: update
+published: true
+rss: true
+collection: updates
+category: chrome
+product: chrome
+type: news
+date: 2015-07-22
+
+title: "MediaStream deprecations"
+description: "The MediaStream API represents synchronized streams of audio or video. MediaStream.ended, MediaStream.label and MediaStream.stop() are being deprecated. Use MediaStream.active, MediaStreamTrack.label and MediaStreamTrack.stop() instead."
+article:
+  written_on: 2015-07-22
+  updated_on: 2015-07-22
+authors:
+  - samdutton
+tags:
+  - audio
+  - video
+  - media
+  - MediaStream
+  - MediaStreamTrack
+  - getUserMedia()
+  - RTCPeerConnection
+  - WebRTC
+permalink: /updates/2015/07/mediastream-deprecations.html
+---
+<style type="text/css" media="screen">
+img, video {
+  max-width: 100%;
+}
+</style>
+
+If you work with `getUserMedia()` or WebRTC, you may need to adjust your code
+for Chrome 45 and later.
+
+The [MediaStream API](http://www.w3.org/TR/mediacapture-streams/) represents
+synchronized streams of media. For example, a stream taken from camera and
+microphone input has synchronized video and audio tracks. Each track is
+represented by a
+[MediaStreamTrack](http://www.w3.org/TR/mediacapture-streams/#mediastreamtrack).
+(Not to be confused with the [&lt;track&gt;
+element](http://www.html5rocks.com/en/tutorials/track/basics/)!)
+
+There are three `MediaStream` deprecations in Chrome 45:
+
+* `MediaStream.ended`
+* `MediaStream.label`
+* `MediaStream.stop()`
+
+In parallel are two new additions:
+
+* `MediaStream.active`
+* `MediaStreamTrack.stop()`
+
+These require the following changes:
+
+* Use `MediaStream.active` to check if a `MediaStream` is actually
+  streaming, not `MediaStream.ended`.
+* Use `MediaStreamTrack.stop()` to stop streaming, not `MediaStream.stop()`.
+* As ever, `MediaStream.id` provides a unique identifier for a `MediaStream`
+  (no change there) and `MediaStreamTrack.label` provides a human-readable
```

Comment:
"As ever...(no change there)." This is redundant. If you're trying to say something else, please reword.